src/main/java/org/example/userinterface/CommandCentre.java

```diff
@@ -69,49 +105,79 @@ private void createPlateau() {
         }
     }
 
-    private void dropRover() {
+    private void dropVehicle() {
         while (true) {
             System.out.println("Input vehicle drop site. (Format: X Y D)");
             System.out.println("X -> X Coordinate");
             System.out.println("Y -> Y Coordinate");
             System.out.println("D -> Cardinal compass directions -> N/E/S/W");
             try {
                 Position pos = SetupInputParser.createInitialPosition(scanner.nextLine());
-                Vehicle rover = new Rover(pos);
-                plateau.landVehicle(rover);
-                this.activeVehicle = rover;
-                System.out.println("Vehicle type: " + rover.getClass().getSimpleName() + " launched at " + rover.reportPosition());
+                int vehicleType = specifyVehicleType();
+                Vehicle vehicle = vehicleBuilder(vehicleType, pos);
```

Comment:
vehicle builder could become a separate class following the Builder design pattern